Makefile

```diff
@@ -85,7 +85,7 @@ coverage:
 		rm -f coverage-tmp.txt;  \
 		$(GO) test -coverprofile=coverage-tmp.txt $$p ; \
 		if [ -f coverage-tmp.txt ]; then \
-			cat coverage-tmp.txt |grep -v 'mode:' >> coverage.txt; \
+			cat coverage-tmp.txt |  grep -v 'mode:' >> coverage.txt || /bin/true; \
```

Comment:
Is it two spaces better than 0? :P

Reply:
haha, shite :see_no_evil:

Reply:
https://github.com/mendersoftware/mender-cli/pull/60